.idea/runConfigurations.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't remove unrelated files to your PR.

Could you revert this?

Reply:
sorry by mistake, done